# STag parameters
libraryHD: 15
errorCorrection: 7
tagSize: 0.163

# Logitech USB configuration
camera_info_topic: /usb_cam/camera_info
raw_image_topic: /usb_cam/image_raw
is_compressed: true

# node configuration

tag_config_json: /home/brennan/Documents/melodic_ws/src/stag/cfg/tag_config.json
tag_tf_prefix: stag_

# Publish debug images
debug_images: true